packages/store/src/internal/state-operations.ts

```diff
@@ -12,12 +12,7 @@ import { deepFreeze } from '../utils/freeze';
  */
 @Injectable()
 export class InternalStateOperations {
-
-  constructor(
-    private _stateStream: StateStream,
-    private _dispatcher: InternalDispatcher,
-    private _config: NgxsConfig
-  ) {
+  constructor(private _stateStream: StateStream, private _dispatcher: InternalDispatcher, private _config: NgxsConfig) {
```

Comment:
Please revert this change. Seems like your formatter got too eager.

Reply:
It's pretty strange considering I have no globally installed libraries and this change was made automatically before commit (commited via tty).

Reply:
When I run prettier in your project on this file, the output is:
`constructor(private _stateStream: StateStream, private _dispatcher: InternalDispatcher, private _config: NgxsConfig) {`
This file is changed automatically when husky runs prettier on whole project before commit.

I can try to force-change it via github to omit husky, but should I do it?